.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: python
+
+sudo: false
+
+python:
+  - "3.5"
+  - "3.6"
+
+install:
+  - pip install -U -r requirements.txt
+  - pip install pytest
```

Comment:
`pytest` should be in `deps`. 
Once again, it is a standalone. We can have every possible dependency in `requirements.txt`.